### ISSUE TITLE
Hardware Decoding not available on Parsec

### ISSUE THREAD

**Issue by M4th12** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#859]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Parsec

### Processor / Processor Number [Required]

AMD Ryzen 5500

### Graphic Card [Required]

Intel ARC A750

### GPU Driver Version [Required]

32.0.101.6130_101.6048

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [X] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [ ] Not applicable

### Windows Build [Required]

Windows 10 22H2

### Intel System Support Utility report

[report.txt](https://github.com/user-attachments/files/17606928/report.txt)

### Description and steps to reproduce [Required]

Open Parsec and connect with another PC. Parsec will use Software decoding instead of Hardware. You can't even select the Hardware option
![image](https://github.com/user-attachments/assets/bd45bc58-8890-4eda-8b60-e18c5f1cfb7f)

**Comment by M4th12** [comment]
I've already try to DDU the driver and unistalling and reinstalling Parsec multiple time. This problem persist even with previous driver version

**Comment by Felipe-Intel** [comment]
Hello @M4th12 
I just made a quick test and install Parsec and the option is available for me. ⁠
![image](https://github.com/user-attachments/assets/774bfc3c-0666-47a6-ba79-f29c17c5e060)
I checked in a system with Ryzen 5 5800X with A750, driver 32.0.101.6130. 
Also check in a 13600K with A750, driver 6130 as well and no issues either. 
The only difference is I have windows 11 23H2. I'll prepare a system with Win10, but given it works in Win11, it may not be a driver issue. 
Have you tried uninstalling/reinstalling Parsec after the driver installation?

**Comment by M4th12** [comment]
@Felipe-Intel yes I reinstalled Parsec multiple times. Only Parsec has this problem other software like obs and other don't have this issue

**Comment by Felipe-Intel** [comment]
Hello @M4th12 just made the test on win10 and the option is not there, so the problem may be related to that specific OS. I'll do a couple more tests over win10 and consult with our driver development team if this is something fixable via driver or if it is on the app side. Have you tried to reach Parsec support? Not sure if there's any. Parsec's Discord channel has a "#help" section.

**Comment by M4th12** [comment]
> Hello @M4th12 just made the test on win10 and the option is not there, so the problem may be related to that specific OS. I'll do a couple more tests over win10 and consult with our driver development team if this is something fixable via driver or if it is on the app side. Have you tried to reach Parsec support? Not sure if there's any. Parsec's Discord channel has a "#help" section.

Ok, thanks I will try to contact them

**Comment by Felipe-Intel** [comment]
I just submitted this issue to our drivers team. I'll let you know any news as soon as I have any.

**Comment by M4th12** [comment]
> I just submitted this issue to our drivers team. I'll let you know any news as soon as I have any.

Ok, in the meantime I submitted the issue to the Parsec Team

**Comment by Felipe-Intel** [comment]
hey @M4th12 I was informed that there are other app behaving similarly (i.e. working in Win11 but not in Win10) and it needs to be investigated along with Microsoft. I asked to add Parsec to this testing but wanted to let you know because it involves a 3rd party (MSFT) so it might take more than expected to get a fix. 
Quick fix would be to update to Win11, but knowing this is not an option for some users for specific reasons, just wanted to point that out.

**Comment by SavageWaffle** [comment]
Is this resolved then? Issue seems tied to Windows 10 users only and would need to be fixed by app devs?

**Comment by M4th12** [comment]
> Is this resolved then? Issue seems tied to Windows 10 users only and would need to be fixed by app devs?

I've already sent the problem to Parsec Devs but the problem persist even now. In the meantime I switched to Windows 11 to get around the problem

**Comment by Felipe-Intel** [comment]
@SavageWaffle This is still under investigation.

**Comment by Felipe-Intel** [comment]
Hello @M4th12 I'm not seeing this issue anymore with our latest driver 32.0.101.6632. If you still have the chance to test on Win10, I'd appreciate your feedback. If not, I will close this ticket.
